build-tools/tasks/generate-icons.js

```diff
@@ -5,13 +5,6 @@ const { src, dest, parallel } = require('gulp');
 const themes = require('../utils/themes');
 const { through, task } = require('../utils/gulp-utils');
 
-// taken from webpack raw-loader: https://github.com/webpack-contrib/raw-loader/blob/master/src/index.js#L14
-function escapeContent(source) {
-  return JSON.stringify(source)
-    .replace(/\u2028/g, '\\u2028')
-    .replace(/\u2029/g, '\\u2029');
```

Comment:
These lines were taking care of preserving newlines when serializing SVG content. Not relevant in JSX use-case anymore